stdlib/2/os/__init__.pyi

```diff
@@ -10,7 +10,7 @@ from mypy_extensions import NoReturn
 error = OSError
 name = ...  # type: str
 
-class _Environ(MutableMapping[str, str]):
+class _Environ(MutableMapping[Union[str, unicode], Union[str, unicode]]):
```

Comment:
Hmm, that's different from what I see. I can actually get `unicode` objects out of `os.environ`. I wonder what's going on.

A potential way to work around this is to define custom `__getitem__` and `__setitem__`  overrides in `_Environ`. They would accept `Union[str, unicode]` arguments but `__getitem__` would always return `str`.